monero-GUI-guide.md

```diff
@@ -66,6 +82,23 @@ Windows Preparation
 If you are on Windows:
 
 -   Make sure that your antivirus does not block the program.
+
+A miner is included in the Monero GUI software and, as a result, most
+anti virus software flags the Monero GUI files as malware and
+subsequently quarantines them. Fortunately, however, there's a fairly
+trivial work around you can use. That is:\
++ Create a new directory / folder.\
++ Open your AV software.\
++ Add an exception for the newly created directory / folder. Put
+differently, you have to whitelist the newly created directory /
+folder.\
```

Comment:
idk where those `\` are comming from: maybe trailing 4*spaces?

Reply:
It's caused by the 2*spaces. They are not needed in a list